### PR TITLE
init model

### PR DESCRIPTION
**Pull request by SJS-Crystal**

### CODE REVIEW

**Comment by SJS-Crystal** [comment]
ready

File: app/Task.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $fillable = ['name'];
```
Comment (quoctd-0607) [inline app/Task.php]: protected $fillable = [
'name',
];

**Comment by SJS-Crystal** [comment]
ready

**Comment by quoctd-0607** [comment]
merged

**Pull request merged**